jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -246,15 +266,61 @@ def _on_rs_redshift_updated(self, event):
         self.rs_rv = self._redshift_to_velocity(value)
         self._rs_disable_observe = False
-
         self._update_line_positions()
 
         if not self._rs_pause_tables:
+            # TODO: try to avoid essentially repeating the loop from above, careful to minimize
+            # updates to vue, maybe pause traitlets?
+            new_list_contents = {}
+            for list_name, line_list in self.list_contents.items():
+                for i, line in enumerate(line_list['lines']):
+                    if self._rs_line_obs_change[0] == list_name and self._rs_line_obs_change[1] == i:  # noqa
+                        # this trigger is coming from a manual change to the observed
+                        # wavelength and would result in a small change to the value before the
+                        # user can finish typing.  So we'll just keep the old value until the
+                        # widget is blurred (loses focus)
+                        line_list['lines'][i]['obs'] = self._rs_line_obs_change[2]
+                    else:
+                        line_list['lines'][i]['obs'] = self._rest_to_obs(float(line['rest']))
+
+                new_list_contents[list_name] = line_list
+
+            self.list_contents = {}  # TODO: fix with dict magic thing
+            self.list_contents = new_list_contents
+
             # Send the redshift back to the Specviz helper (and also trigger
             # self._update_global_redshift)
-            msg = RedshiftMessage("redshift", self.rs_redshift, sender=self)
+            msg = RedshiftMessage("redshift", value, sender=self)
             self.app.hub.broadcast(msg)
 
+    def vue_change_line_obs(self, kwargs):
+        # NOTE: we can only pass one argument from vue (it seems), so we'll pass as
+        # a dictionary (kwargs) instead of positional or keyword arguments (**kwargs)
+        line_obs = kwargs.get('obs_new')
+        if isinstance(line_obs, str) and not len(line_obs):
+            # empty string, we don't want to revert yet because then
+            # the user can never delete the entry and type something new
+            # so we'll just leave empty
+            return
+        list_name = kwargs.get('list_name')
+        line_ind = kwargs.get('line_ind')
+        line = self.list_contents[list_name]['lines'][line_ind]
+        line_rest = float(line['rest'])
+        if line_obs is None:
+            # then coming from the blur, we'll keep the latest update from the @change
+            line_obs = float(line['obs'])
+
+        # we don't want this call to recursively update THIS obs wavelength, but DO want it to
+        # update the RV and all other obs wavelengths.  Once tabbing or losing focus, vue will
+        # send another event with avoid_feedback=False so that the wavelength updates to
+        # exactly match the redshift (so that can be considered the ground truth value consistently)
+        if kwargs.get('avoid_feedback', False):
```

Comment:
Maybe store `kwargs.get('avoid_feedback', False)` in a var, since it is called again below?